app/models/user.rb

```diff
@@ -1,5 +1,14 @@
 class User < ActiveRecord::Base
   has_secure_password
 
+  require 'random_hash'
+
   enum role: [ :admin, :curator, :cataloguer ]
+
+
+  def self.create_invited_user(user_params)
```

Comment:
Can you rename this method? It could be confusing because `ActiveRecord::create` will create a record in the database, but this method doesn't.